cfgov/unprocessed/apps/owning-a-home/js/explore-rates/Slider.js

```diff
@@ -0,0 +1,113 @@
+// Required modules.
+import * as atomicHelpers from '../../../../js/modules/util/atomic-helpers';
+import * as standardType from '../../../../js/modules/util/standard-type';
+
+const $ = require( 'jquery' );
+
+import { sliderLabel } from './template-loader';
+import * as domValues from './dom-values';
+import 'rangeslider.js';
+
+/**
+ * Footer
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the organism.
+ * @returns {Footer} An instance.
+ */
+function Slider( element ) {
+
+  const BASE_CLASS = 'a-range';
+  const _dom = atomicHelpers.checkDom( element, BASE_CLASS );
+  let _min;
+  let _max;
+  let _onSlideEndHandler;
+  let _sliderTextDom;
+
+  /**
+   * @param {number} min - Minimum value of the range.
+   * @param {number} max - Minimum value of the range.
+   * @param {HTMLNode} sliderTextDom - HTML element used for slider range text.
+   * @param {Function} onSlideEndHandler - Callback when user releases slider.
+   * @returns {Slider|undefined} An instance,
+   *   or undefined if it was already initialized.
+   */
+  function init( min, max, sliderTextDom, onSlideEndHandler ) {
+    if ( !atomicHelpers.setInitFlag( _dom ) ) {
+      return standardType.UNDEFINED;
+    }
+
+    _min = min;
+    _max = max;
+    _onSlideEndHandler = onSlideEndHandler;
+    _sliderTextDom = sliderTextDom;
+
+    return this;
+  }
+
+  /**
+   * @returns {number} Minimum value of the range.
+   */
+  function getMin() {
+    return _min;
+  }
+
+  /**
+   * @returns {number} Maximum value of the range.
+   */
+  function getMax() {
+    return _max;
+  }
+
+  /**
+   * Initialize the range slider. http://andreruffert.github.io/rangeslider.js/
+   */
+  function render() {
+    $( '#' + element.id ).rangeslider( {
```

Comment:
I would check if there is a range slider which doesn't include jQuery

Reply:
Hm, I'll be. Lookie lookie https://github.com/stbaer/rangeslider-js. I'll give it a try.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/3927/commits/3fed220c49f4d482c67bbc0dccc61bf4e953fa3e

Reply:
Err, hang on. I needed to `gulp clean` and now find it's not working as expected.